home/ivp-planning/advanced-tools/video-export-tool.md

```diff
@@ -0,0 +1,10 @@
+# Video Export
+
+With update 2307.0 it is now possible to directly export your recordings via the "Video Export" option, found in the "File" drop-down menu in the top left of the program.
+
+![](../../../.gitbook/assets/video-export.png)  
+
+Once you have clicked on the "Video Export" option, a new window will open, in which you can make changes to your recording.  
```

Comment:
Regarding the writing style of this documentation, please consider the following:

1. Avoid using past time
2. Keep sentences short and simple/precise

```suggestion
Click on "Video Export" to open a new window that allows you to configure your recording.
```